rules/integrations/azure/credential_access_azure_key_vault_retrieval_from_rare_identity.toml

```
@@ -0,0 +1,142 @@
+[metadata]
+creation_date = "2025/07/10"
+integration = ["azure"]
+maturity = "production"
+updated_date = "2025/07/10"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies secrets, keys, or certificates retrieval operations from Azure Key Vault by a user principal that has not
+been seen doing so in the last 10 days. Azure Key Vault is a cloud service for securely storing and accessing secrets,
```

```suggestion
been seen doing so in the last 14 days. Azure Key Vault is a cloud service for securely storing and accessing secrets,
```
Ruben always suggests removing the time frame from the description since we often change the time window as part of tuning and may forget to change the description to match.